ovs-dpdk.md

```diff
@@ -0,0 +1,180 @@
+# oVirt now supports OVS-DPDK
+
+
+[DPDK](http://dpdk.org/) (Data Plane Development Kit) provides high-performance packet processing libraries and user space drivers.
+[Open vSwitch](http://openvswitch.org/)  supports DPDK since v2.2 and starting with version 2.4, a DPDK-optimized vHost path is enabled in OVS.
+ 
+According to [Intel ONP performance tests](https://download.01.org/packet-processing/ONPS2.1/Intel_ONP_Release_2.1_Performance_Test_Report_Rev1.0.pdf), using OVS with DPDK can provide a huge performance enhancement, increasing network packets throughput and reduced latency.
+ 
+OVS-DPDK has been added to oVirt as an experimental feature.
+Please note that OVS-DPDK cannot access any features located within the Linux kernel, for example: Linux bridge,  tun/tap devices, iptables etc.
+
+## Requirements:
+
+Please follow the instructions in <http://dpdk.org/doc/guides-16.04/linux_gsg/nic_perf_intel_platform.html> in order to acheive best performance.
```

Comment:
Suggest:
"In order to achieve the best performance, please follow the instructions at: 
<http://dpdk.org/doc/guides-16.04/linux_gsg/nic_perf_intel_platform.html>"   (code on new line)